java/maven/nbproject/project.xml

```diff
@@ -369,7 +369,7 @@
                     <compile-dependency/>
                     <run-dependency>
                         <release-version>1</release-version>
-                        <specification-version>1.4</specification-version>
+                        <specification-version>1.45</specification-version>
```

Comment:
Looks like a large jump, should this be `1.5`?

Reply:
No - see [change in XAM in this PR](https://github.com/apache/netbeans/pull/3013/files#diff-d4cbe9616caee5c9e8a0b7766f54bf932349c61e0bcc223082bbebfa3c117ae2R24)

Reply:
Thank you - makes sense now.